src/partials/_top11_save.php

```diff
@@ -44,16 +75,23 @@
 
 // Save the vote
 try {
-    // Record that this user has voted this week (prevent duplicates)
-    $top11Model->recordUserVote($voter_email, $auth0_id);
-    
-    // Extract user information from email for contestant entry
-    $emailParts = explode('@', $voter_email);
-    $username = (count($emailParts) === 2) ? $emailParts[0] : $voter_email;
-    
-    // Save contestant info using the authenticated email
-    if ($contest === 'yes' || $newsletter === 'yes') {
-        $top11Model->addContestant($username, '', $voter_email, '', $contest, $newsletter);
+    if ($useAuthVoting) {
+        // Auth voting mode: record user vote to prevent duplicates
+        $top11Model->recordUserVote($voter_email, $auth0_id);
+        
+        // Extract user information from email for contestant entry
+        $emailParts = explode('@', $voter_email);
+        $username = (count($emailParts) === 2) ? $emailParts[0] : $voter_email;
+        
+        // Save contestant info using the authenticated email
+        if ($contest === 'yes' || $newsletter === 'yes') {
+            $top11Model->addContestant($username, '', $voter_email, '', $contest, $newsletter);
```

Comment:
I think we're still going to want to collect the user's personal information - first name, last name, phone number. So there will still need to be a small form